Assets/Scripts/Game.cs

```diff
@@ -72,6 +75,13 @@ void Start()
         // Create some things
         uiScaler = new UIScaler(uICanvas);
         config = new ConfigFile();
+        currentLang = 1; // English
+
+        if (config.data.Get("MoMConfig") != null)
```

Comment:
Actually, why is the language in MoMConfig anyway?  I would expect all game types to share language settings.